includes/Admin/Products.php

```diff
@@ -88,7 +108,7 @@ public static function render_commerce_fields( \WC_Product $product ) {
 			<input type="checkbox" class="enable-if-sync-enabled"
 			       name="<?php echo esc_attr( self::FIELD_COMMERCE_ENABLED ); ?>"
 			       id="<?php echo esc_attr( self::FIELD_COMMERCE_ENABLED ); ?>" value="yes"
-			       checked="<?php echo \SkyVerge\WooCommerce\Facebook\Products::is_commerce_enabled_for_product( $product ) ? 'checked' : ''; ?>">
+			       checked="<?php echo Products_Handler::is_commerce_enabled_for_product( $product ) ? 'checked' : ''; ?>">
```

Comment:
can also use `<?php checked( Products_Handler::is_commerce_enabled_for_product( $product ) ); ?>` here

Reply:
I think we should use `checked()`. The `checked` HTML attribute [is boolean](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#checked), so if present (regardless of the value) the field will be checked by default.

Reply:
good catch if echoing should echo `checked="checked"` or nothing at all